### ISSUE TITLE
workaround for staleness (outdated content) when e.g. Firefox is used to view GitHub issues

### ISSUE THREAD

**Issue by grahamperrin** [sindresorhus/refined-github#672]
## Enhancement request

Might _Refined GitHub_ gain a preference to somehow ignore/override GitHub's caching policy? 

## Background

From https://bugzilla.mozilla.org/show_bug.cgi?id=706970#c75 under Mozilla bug [706970 - Tabs restored at startup (app tabs, session restore) should be refreshed from server, not in a "stale" state from cache](https://bugzilla.mozilla.org/show_bug.cgi?id=706970):

> … The staleness with GitHub can be particularly troublesome; it's commonplace for me to find things two or more days out of date … 

## An example of the problem

Three screenshots, in chronological order: 

### 2017-08-13 15:46:46

I knew immediately that this was wrong, because what appears to be drafted in the _Write_ field had been posted by me hours earlier: 

![2017-08-13 15 46 46](https://user-images.githubusercontent.com/192271/29396336-382a3362-830f-11e7-81ca-a377c51a44fe.png)

### 2017-08-13 15:47:37

Following a reload of the page, there it was, my post from five hours earlier: 

![2017-08-13 15 47 37](https://user-images.githubusercontent.com/192271/29396343-47f48b3a-830f-11e7-8d4d-cf7d3684cebb.png)

Context: `https://github.com/mozilla/addons/issues/468#event-1202478995`

### 2017-08-13 15:48:29

After scrolling down, there it was again, the draft of my post from five hours earlier, misrepresented in the _Write_ field below the successful post: 

![2017-08-13 15 48 29](https://user-images.githubusercontent.com/192271/29396349-50776822-830f-11e7-8475-ecfef2e8b393.png)

In such cases, where Firefox misrepresents a draft of a post that is already posted, I typically: 

1. select everything in the draft
2. <kbd>Delete</kbd>
3. <kbd>Control</kbd>-<kbd>F5</kbd> reload the page to minimise the likelihood of further misrepresentation.

**Comment by bfred-it** [comment]
I don't think this should be part of RGH (if an extension can mess with Firefox' own caching at all)

**Comment by grahamperrin** [comment]
Yeah, drop this idea. I was clutching at straws. 

tl;dr re the conversation that followed <http://logs.glob.uno/?c=mozilla%23firefox&s=13+Sep+2017&e=13+Sep+2017#c450899> I no longer suspect a cache-related problem. More likely a bug with, or affecting, session storage.

**Comment by grahamperrin** [comment]
… and the problems with GitHub are not limited to Firefox. My previous comment, submitted with SeaMonkey, was non-apparent after I closed the tab then reopened the recently closed tab.
